shinylive/_deps.py

```diff
@@ -120,26 +122,51 @@ def _pyodide_pkg_infos_to_quarto_html_dep_items(
 def shinylive_base_deps_htmldep(
     sw_dir: Optional[str] = None,
     *,
-    all_files: bool = False,
+    file_type: AssetFileType = ("base",),
```

Comment:
```suggestion
    asset_type: AssetType = ("base",),
```

Reply:
When implemented, flake8 gives:
> Do not use mutable data structures for argument defaults.  They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.Flake8(B006)

Using Tuple approach instead of Set.